test/DynamoCoreTests/PackageManager/PackageManagerClientTests.cs

```diff
@@ -33,7 +33,7 @@ public void ListAll_ReturnsNonEmptyListWhenPackagesAreAvailable()
                 x.ExecuteAndDeserializeWithContent<List<PackageHeader>>(It.IsAny<HeaderCollectionDownload>()))
                 .Returns(mpl);
 
-            var m = new PackageManagerClient(c.Object, MockMaker.Empty<IPackageUploadBuilder>());
+            var m = new PackageManagerClient(c.Object, MockMaker.Empty<IPackageUploadBuilder>(), "");
```

Comment:
We could have a style discussion here about "" vs. `string.Empty`, but the internet seems to be split on the subject.